<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XTConvertUI</class>
 <widget class="QDialog" name="XTConvertUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XTConvert</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../_resources/maintainer.qrc">
    <normaloff>:/maintainer.ico</normaloff>:/maintainer.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Here you can package an asset pack or episode for TheXTech&lt;br/&gt;to share it with others or play it on low-end consoles.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Content:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="content_path">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_content">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Content type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="content_type">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="hr1">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Target platform:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="target_platform">
         <item>
          <property name="text">
           <string>Mainline</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3DS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wii</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Output:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="output_path">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_output">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QProgressBar" name="progress">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string notr="true">Start!</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textBrowser_3">
        <property name="readOnly">
            <bool>true</bool>
        </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../_resources/maintainer.qrc"/>
 </resources>
 <connections/>
</ui>
